docs/DeveloperGuide.md

```diff
@@ -318,6 +318,44 @@ The following activity diagram summarizes what happens when a user executes a ne
   * Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
   * Cons: We must ensure that the implementation of each individual command are correct.
 
+
+### Setting a Default Group for a particular Mod
+
```

Comment:
Please include this after the `###` header
`[<sub><sup>Back to top</sup></sub>](#table-of-contents)`
to link back to the top/table of contents!

Reply:
sure:))

Reply:
Not sure if it's a bug or anything but I dont see change in the code nor the generated page? Has this been done?

Reply:
My fault again, I didn't realise that this was for the DG - thanks!